sormas-backend/src/main/java/de/symeda/sormas/backend/sormastosormas/SormasToSormasFacadeEjb.java

```diff
@@ -646,7 +669,7 @@ private void sendEntityToSormas(Object entity, String endpoint, SormasToSormasOp
 		if (statusCode < 200 || statusCode >= 400) {
```

Comment:
is there anything besides 200 that is acceptible here?

Reply:
for now, not really but maybe in the future. For example 201, 203 are widely used on defined rest API-s